kolibri/plugins/setup_wizard/assets/src/views/index.vue

```diff
@@ -1,33 +1,74 @@
 <template>
 
-  <div class="device-owner-creation">
+  <div class="setup">
     <div class="wrapper" role="main">
       <img class="logo" src="./icons/logo-min.png" alt="Kolibri logo">
-      <div class="container">
-        <h1>{{ $tr('header') }}</h1>
-        <h2 class="title">{{ $tr('deviceOwner') }}</h2>
-        <div class="description">{{ $tr('deviceOwnerDescription') }}</div>
-        <div class="creation-form">
-          <p class="error-message" role="alert" aria-atomic="true">{{ errormessage }}</p>
-          <label for="nameinput" class="inputlabel">{{ $tr('username') }}:</label>
-          <input id="nameinput" :class="{ 'input-error': username_error }" type="text" v-model="username">
-          <label for="passwordinput" class="inputlabel">{{ $tr('password') }}:</label>
-          <input id="passwordinput" :class="{ 'input-error': password_error }" type="password" v-model="password">
-          <label for="confirminput" class="inputlabel">{{ $tr('confirmPassword') }}:</label>
-          <input id="confirminput" :class="{ 'input-error': password_error }" type="password" v-model="confirm_password">
-        </div>
-        <h2 class="title">{{ $tr('facility') }}</h2>
-        <div class="description">{{ $tr('facilityDescription') }}</div>
-        <label for="facilityinput" class="inputlabel">{{ $tr('facilityName') }}:</label>
-        <input id="facilityinput" :class="{ 'input-error': facility_error }" type="text" v-model="facility">
-        <br>
-        <br>
-        <br>
-        <div class="btn-wrapper">
-          <button class="create-btn" type="button" @click="createBoth">{{ $tr('getStarted') }}</button>
-        </div>
-      </div>
-      <br>
+
+
+      <form @submit.prevent="submitSetupForm" novalidate class="container">
```

Comment:
Added `novalidate` so that native error styling wouldn't interfere with the global error message I've set up for empty fields. I'm iffy on this, so I'll revert that commit if you all think it's a bad idea.